tests/testthat/test-f7Accordion.R

```diff
@@ -65,9 +65,9 @@ test_that("accordion works as expected", {
     name = "accordion-app",
     variant = platform_variant(),
   )
-  app$expect_values()
+  app$expect_values(input = "myaccordion1")
   app$click(selector = "#go")
   # Animation/transition takes a bit of time
   app$wait_for_idle(1000)
-  app$expect_values()
+  app$expect_values(input = "myaccordion1")
 })
```

Comment:
@DivadNojnarg since I added back in app.js, sessionId and device info are also stored in the environment. If we would only use `expect_values()`, this will all be snapshotted too- and will never be the same. So I suggest being very explicit in what inputs/outputs/exports to expect

Reply:
Yes I had this issue when I intialized the test. I would say we may not support this anymore (sessionId), as nobody use it very likely.

Reply:
I think `deviceInfo` will also cause a problem with testing. This is the comment there: `Framework7.device is extremely useful to set up custom design` - so I guess that's handy to have. To make the tests more future proof, I think being explicit now will save us some trouble later. Who knows what we might add to the environment 📦